src/config/select.js

```diff
@@ -1,4 +1,3 @@
-import React from 'react';
```

Comment:
Keep in mind even if we don't use React itself we need it in every file where we use JSX. Simply because JSX gets compile with Babel to function calls e.g. `React.DOM.ul()`

https://facebook.github.io/react/docs/glossary.html#factories